### DISCUSSION TITLE
Feature Request: Limited User Permissions for Individual Mail Access

### DISCUSSION THREAD

**Discussion by captainbanan** [postalserver/postal#3224] [Feature suggestions]
Dear Postal Development Team,

I am a user of Postal and would like to suggest a feature that could enhance the flexibility and usability of the platform. Specifically, I am interested in a way to set up limited permissions for users, enabling them to view only their own emails and use the server as a regular mailbox, without access to other users' data or administrative functions.

This feature would allow me to provide my users with basic access while maintaining the privacy and security of each account. If this capability already exists, I would appreciate guidance on configuring it. Otherwise, I believe such functionality would be a valuable addition to the software for users with similar needs.

Thank you for considering this request, and I look forward to your response.

**Comment by willpower232** [comment]
Postal isn't really intended for long term storage and searching of messages, you would do better implementing this in whichever system is sending messages to Postal. You can keep the data for as long as you want, manage access however you want, and use webhooks to keep updated with what happened to the messages.
